### ISSUE TITLE
SPVisioServiceApp is not creating the proxy

### ISSUE THREAD

**Issue by wickedave** [PowerShell/SharePointDsc#772]
Hi,

I'm trying to configure the Visio Service application and it looks like the service application proxy is not getting created. I don't know if it is just me or if it is really a bug.

Here is part of the error in the log:
```
VERBOSE: [SERVER]:                            [[SPVisioServiceApp]VisioService-01] Leaving BeginProcessing Method
of New-SPVisioServiceApplication.
The timer job completed, but failed on one or more machines in the farm.
    + CategoryInfo          : InvalidData: (Microsoft.Offic...viceApplication:) [], CimException
    + FullyQualifiedErrorId : Microsoft.Office.Visio.Server.Cmdlet.NewSPVisioServiceApplication
    + PSComputerName        : server
```

The timer job info is:
`An update conflict has occurred, and you must re-try this action. The object VisioGraphicsServiceApplication Name=Lab-Visio-01 was updated by domain\username, in the wsmprovhost (2208) process, on machine SERVER`

Do you have the same behavior when trying to install it on your side?

The module version on which I'm trying is 2.1.0.0. The SharePoint version is 2016 and OS is Windows Server 2016.

**Comment by wickedave** [comment]
I just did another test and the error is now
`Unable to find the Visio Graphics Service Application with name VisioGraphicsServiceApplication Name=Lab-Visio-01`

It's like the proxy is starting to be created before service application is fully created. When looking at the service application, I can see the Visio Service application without a proxy.

**Comment by NikCharlebois** [comment]
@wickedave I am getting the same issue. Currently investigating the source

**Comment by NikCharlebois** [comment]
Looks like the name is not properly passed to the New-SPVisioServiceApplicationProxy cmdlet internally. This issue was most likely introduced back in 1.6.0.0 when we changed the way we created proxy. It will be fixed in my next PR. Thanks

**Comment by ykuijs** [comment]
Closed via PR #772
